torch/csrc/api/include/torch/nn/functional/linear.h

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+#include <torch/types.h>
+
+namespace torch {
+namespace nn{
+namespace functional {
+
+inline Tensor bilinear(const Tensor& input1, const Tensor& input2, const Tensor& weight, const Tensor& bias) {
```

Comment:
In the Python version, `bias` can be `None`, and to do the same we would need `const Tensor& bias = Tensor()` here :D